pacman-nodejs-app/tools/migrate/kmt.sh

```diff
@@ -215,12 +220,15 @@ function verify_services_ready {
         for s in ${services}; do
             # Filter service load balancer IP address
             local service_ip=$(kubectl get service ${s} -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+	    if [[ ${service_ip} == '' ]]; then
+                 local service_ip=$(kubectl get service ${s} -o jsonpath='{.status.loadBalancer.ingress[0].hostname}')
+            fi
```

Comment:
I know it does look off, and even the indentation of all_ready and break now look like they are off by one space but if I press the edit button to change it online (even!), the indentation is fine and git doesn't accept attempting to realign it as having changed (because there isn't any change). I don't know what else to do to fix this